docker/src/main/docker/postgresql/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2'
+services:
+  avatica:
+    ports:
+      - "8765:8765"
+    links:
+      - postgresql
+    image: avatica-postgresql-server
+    command: "-u 'jdbc:postgresql://postgresql:5432/avatica' -p 8765"
+    depends_on:
+      - postgresql
+  postgresql:
+    image: postgres:9.6
+    ports:
+      # Default port for the docker image
+      - "5432:5432"
```

Comment:
Since postgres is designed to be behind avatica, it may be possible to just set this to "5432" and not bind it to 5432 locally as well. You could get the postgres port with `docker-compose ps` and a few other ways. Otherwise this would conflict with postgresql already running.